protocol/protobuf/application_metadata_message.proto

```diff
@@ -79,9 +79,10 @@ message ApplicationMetadataMessage {
     SYNC_ENS_USERNAME_DETAIL = 64;
     SYNC_ACTIVITY_CENTER_NOTIFICATION = 65;
     SYNC_ACTIVITY_CENTER_NOTIFICATION_STATE = 66;
-    COMMUNITY_ADMIN_MESSAGE = 67;
+    COMMUNITY_EVENTS_MESSAGE = 67;
```

Comment:
I think now you can safely remove
`repeated bytes events = 3 [deprecated=true];` from protobuf

Reply:
Not sure I am following :thinking: Why?

Reply:
The previous protobuf can be recognized by the clients only if MessageType is COMMUNITY_ADMIN_MESSAGE
Taking into account, that you changed MessageType, your clients will not handle COMMUNITY_ADMIN_MESSAGE, they'll handle COMMUNITY_EVENTS_MESSAGE, and this MessageType is new in the network and the network doesn't know anything about the previous `protobuf.CommunityEventsMessage` format

Reply:
This change  is only local, it's just renaming a variable, but has not impact on what's passed over the wire, @osmaczko to confirm just to make sure, so protocol-wise everything is the same

Reply:
ah... sorry, we will send an integer...

Reply:
Fields in a protobuf message are identified by field numbers, not by field names. Changing the name of the message does not have any impact, it is still the same message.